doc/api/stream.md

```diff
@@ -1817,6 +1823,57 @@ const myWritable = new Writable({
 });
 ```
 
+#### writable.\_construct(options, callback)
```

Comment:
```suggestion
#### `writable._construct(options, callback)`
```

In a recent documentation change, the headers pattern was changed to this one. It would be good to stay consistent but it's not blocking from my side.